### ISSUE TITLE
How do I remove the old authenticator?

### ISSUE THREAD

**Issue by DumbJoe** [Jessecar96/SteamDesktopAuthenticator#790]
![This is going to be a problem](https://user-images.githubusercontent.com/36319788/211123215-50214eea-b927-454e-81b3-0a53365fd6be.png)

**Comment by dyc3** [comment]
If you set it up with the steam mobile app, remove it using the steam mobile app. If you set it up with SDA, remove via SDA. Otherwise, you'll need to use your revocation code, 

If you're just trying to have the same auth codes on multiple devices, just copy your maFiles to the other device. No need to remove and re-add the authenticator.

**Comment by makedir** [comment]
how do you use this? didnt steam remove 3rd party authentication a while ago, the standalone app is not available anymore in plays tore, only the steam app itself. so how does it work?

**Comment by dyc3** [comment]
I'm not sure what you're talking about. Steam never had "3rd party authentication". SDA was made by reverse engineering the steam mobile app.

**Comment by dyc3** [comment]
The steam app is the steam mobile auth app. There was always just one app.

**Comment by makedir** [comment]
What are you even talking about, totally false. there was a separate steam authenticator app.

**Comment by dyc3** [comment]
We're getting off topic here -- the new mobile auth stuff Valve rolled out not too long ago uses the same backend APIs. Additionally, 2fa codes are generated offline, based on the current time. SDA uses these APIs to set up the authenticator, and SDA can just use the secret it has to generate 2fa codes.

**Comment by makedir** [comment]
Thank you that is what I wanted to hear. I need to connect a new steam account with an authentication, and I cant find a tutorial anywhere how it or if it is working with the new steam app. I did this like 10 years ago with the old app, which is not in play store anymore. So how do you do it today, if you dont want to use Steam app, is there a way to use a nother way.

**Comment by dyc3** [comment]
If you are on windows, you can use SDA. I also maintain [steamguard-cli](https://github.com/dyc3/steamguard-cli), which may also work on windows (but you'll have to build it yourself via `cargo install steamguard-cli`).

The releases for SDA can be found on the releases page: https://github.com/Jessecar96/SteamDesktopAuthenticator/releases

**Comment by makedir** [comment]
So I would first use SDA, and then I can I export to WinAuth too or Aegis on Android?

**Comment by dyc3** [comment]
SDA does not support exporting your 2fa secret to another app. You have to take the `uri` field from the `.maFile` that was created, and then create QR code with it, and _maybe_ your other app will generate the correct codes. Google authenticator does not, for example.

**Comment by kkagill** [comment]
but if I remove it, 15 days restriction would start again right?

**Comment by dyc3** [comment]
That is correct. If you remove an authenticator from a steam account, there's a 15 day trade restriction, regardless of if you do it via steam's official mobile app, or otherwise. https://help.steampowered.com/en/faqs/view/451E-96B3-D194-50FC#appremoved

**Comment by DumbJoe** [comment]
> If you set it up with the steam mobile app, remove it using the steam mobile app. 

What if said mobile app no longer allows you to login to do this....?
